### PR TITLE
[BIOMAGE-1911] Make Cellenics code use new S3 buckets

### PR DESCRIPTION
**Pull request by StefanBabukov**
# Description

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1911

#### Link to staging deployment URL (or set N/A)
N/A
#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: src/api/general-services/get-work-results.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const validateTagMatching = async (experimentId, params) => {
 
 const getWorkResults = async (experimentId, ETag) => {
   const params = {
-    Bucket: `worker-results-${config.clusterEnv}`,
+    Bucket: `worker-results-${config.clusterEnv}-${config.accountId}`,
```
Comment (ivababukova) [inline src/api/general-services/get-work-results.js:47]: could you put this one in bucketNames too?

File: src/api/route-services/experiment.js [hunk 2]
```diff
@@ -34,10 +34,10 @@ const logger = getLogger('[ExperimentService] - ');
...
   constructor() {
     this.experimentsTableName = `experiments-${config.clusterEnv}`;
-    this.cellSetsBucketName = `cell-sets-${config.clusterEnv}`;
-    this.processedMatrixBucketName = `processed-matrix-${config.clusterEnv}`;
-    this.rawSeuratBucketName = `biomage-source-${config.clusterEnv}`;
-    this.filteredCellsBucketName = `biomage-filtered-cells-${config.clusterEnv}`;
+    this.cellSetsBucketName = `cell-sets-${config.clusterEnv}-${config.accountId}`;
+    this.processedMatrixBucketName = `processed-matrix-${config.clusterEnv}-${config.accountId}`;
+    this.rawSeuratBucketName = `biomage-source-${config.clusterEnv}-${config.accountId}`;
+    this.filteredCellsBucketName = `biomage-filtered-cells-${config.clusterEnv}-${config.accountId}`;
```
Comment (ivababukova) [inline src/api/route-services/experiment.js:40]: could you import these from `aws/bucketNames`?

File: src/api/route-services/plots-tables.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ const validateRequest = require('../../utils/schema-validator');
 class PlotsTablesService {
   constructor() {
     this.tableName = `plots-tables-${config.clusterEnv}`;
+    this.bucketName = `plots-tables-${config.clusterEnv}-${config.accountId}`;
```
Comment (ivababukova) [inline src/api/route-services/plots-tables.js:9]: could you add this to `aws/bucketNames` and import it from there?

File: src/api/route-services/samples.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ class SamplesService {
   constructor() {
     this.experimentsTableName = `experiments-${config.clusterEnv}`;
     this.samplesTableName = `samples-${config.clusterEnv}`;
-    this.sampleFilesBucketName = `biomage-originals-${config.clusterEnv}`;
+    this.sampleFilesBucketName = `biomage-originals-${config.clusterEnv}-${config.accountId}`;
```
Comment (ivababukova) [inline src/api/route-services/samples.js:22]: could you use this one from `aws/bucketNames`?

**Comment by ivababukova** [comment]
can you print in the logs the s3 buckets where results are fetched from? Just so that we can verify that we are using the right buckets, not the old ones.

**Comment by StefanBabukov** [comment]
> can you print in the logs the s3 buckets where results are fetched from? Just so that we can verify that we are using the right buckets, not the old ones.

I printed the params before getting the signed url for the results: 
image

**Comment by ivababukova** [comment]
> > can you print in the logs the s3 buckets where results are fetched from? Just so that we can verify that we are using the right buckets, not the old ones.
> 
> I printed the params before getting the signed url for the results: image

Can you tell me also what is printed in the logs in staging?

**Comment by StefanBabukov** [comment]
image

**Review by ivababukova — APPROVED** [review]

**Pull request merged**